core/src/main/java/org/apache/brooklyn/core/sensor/ssh/SshCommandSensor.java

```diff
@@ -105,6 +97,7 @@ public void apply(final EntityLocal entity) {
                 Map<String, Object> env = MutableMap.copyOf(entity.getConfig(BrooklynConfigKeys.SHELL_ENVIRONMENT));
 
                 // Add the shell environment entries from our configuration
+                Map<String,Object> sensorEnv = params.get(SENSOR_SHELL_ENVIRONMENT);
```

Comment:
We should probably also do `EntityInitializers.resolve` when retrieving the env.

Reply:
Ignore this comment; later in the code it calls `Tasks.resolveDeepValue(env, ...)`